docs/moderator-handbook.md

```diff
@@ -171,11 +171,17 @@ Forum moderators can delete users' posts. You should only do this for the follow
 
 1. Someone has posted a pornographic or graphically violent image.
 2. Someone has posted a link or code that is malicious in nature and could harm other campers who click on it.
-3. Someone has flooded a thread with a lot of spam messages.
+3. Someone has flooded a thread with a lot of spam messages or is an obvious spam-bot or spammer from the outset (as determined by the pattern of posting once for example then adding a spam link to their post later on to try to defeat the automated spam detection).
 
 ### Dealing with Spam
 
-For the first spam post of a user, send them a message explaining the problem, and remove the link or post as appropriate. Leave a note on the user's profile explaining the action you have taken. If the problem persists, then quietly block the user from posting (using the silence option on the User Admin panel). Send the user a warning with the [Code of Conduct](https://code-of-conduct.freecodecamp.org). Check the box in the private message indicating that your message is a "formal warning."
+For the first spam post of a legitimate user (ie. whose intent isn't to spam the forum but to learn/contribute to the forum), send them a message explaining the problem, and remove the link or post as appropriate. Leave a note on the user's profile explaining the action you have taken. If the problem persists, then quietly block the user from posting (using the silence option on the User Admin panel). Send the user a warning with the [Code of Conduct](https://code-of-conduct.freecodecamp.org). Check the box in the private message indicating that your message is a "formal warning."
+
+#### Some notable situations:
```

Comment:
My main suggestion is to remove this section.

```suggestion
#### Notable Situations:
```

Reply:
I was hoping to add some information here to explicitly call out responsibilities that new mods need to be aware of. (I think for the newest mods, we all share in some confusion of how to handle requests for review, it took a few tries at least and discussion to get insight from other mods on this - frankly there are still some situations that I'm not clear on, so yes, this section is not ideal as-is for what I was hoping for, but we need something to direct new mods to these notable situations (akismet flags, user flags, tricky spammers, etc).

The amount of spam I've clicked on in my first 2 weeks has made me very uncomfortable (because I'm essentially clicking blind to see if this is spam or something else). Calling out the behaviour of adding links after the post is up would have helped me to identify the spammer without having to click a link to try to understand the objective of the post.

Having said that, I'm still not happy with this section. We can defer the update till the new mods and the more experienced mods can agree on what new mods need to know.

Reply:
I hope my suggestions do not come across as harsh; I think updating the mod handbook is an excellent idea.

I think we might need to tackle this from a different standpoint. That is, I suggest just _updating_ the existing sections here, and add a completely new section aimed at _onboarding_ new mods.

Specifically, I would suggest:
1) We should double check the existing docs for out-dated info. Then, remove that info. **No major additions of text**
2) Create a new high-level section `### Onboarding Information`
  - `#### How to Use the Forum` - where to find stuff new (visible) to mods
  - `#### Responsibilities and Expectations`
  - `#### Common Patterns and Actions` - what to expect from bots, spammers, and bad-actors

Ideally, we have:
- _Mod Code of Conduct_ - a guide that tells step-by-step for common situations
- Something less official where most mods will learn by watching. E.g. how to handle solution posters, how to help with support requests, how to handle mis-use of the flag system, how to use canned replies*

\* we should not have a "canned replies" section in the mod book, because this does change. We only have 2/3 canned replies that are "official" (always used in the same situation, following moderation action). Otherwise, most of the canned replies are just "nice to haves" - new mods should take a couple minutes familiarising themselves with what currently exists.

---

That being said, let us wait to hear what other mods think, before putting in the work to adjust this.

Reply:
> The amount of spam I've clicked on in my first 2 weeks has made me very uncomfortable (because I'm essentially clicking blind to see if this is spam or something else). 

For the most part, this is something you end up learning. In the last 2 years, I cannot remember clicking on a single link to "check" if it is spam or not. We are a programming forum - there are only so many domains that are on topic.
There is no requirement for any mod to put themselves in a position they might be uncomfortable with. So, if you are unsure about a link, feel free to flag/reach-out before clicking.

Otherwise, something you can do: Use a different browser/profile with the strictest settings (block images, prevent JavaScript, block all cookies, etc.) to open unknown links.

Reply:
Just a grammar/style note - if you have header markup, you don't need a trailing colon. It also doesn't match the other headers which do not have them.